.claude/skills/pr-flow/SKILL.md

```
@@ -1,350 +1,212 @@
+1. Stash all changes: `git stash`
+2. For each category:
+   - Create branch from develop: `git checkout -b <branch-name> develop`
+   - Restore relevant files: `git stash pop` or selective `git checkout stash -- <files>`
+   - Commit, push, create PR
+3. Track all PR numbers for parallel review monitoring
+
+### Phase 2: Poll for CodeRabbit Review (Non-Blocking)
+
+**DO NOT** use blocking `sleep` loops. Use `CronCreate` for periodic checks:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)